src/components/pages/membership_form/Address.vue

```diff
@@ -2,238 +2,142 @@
 	<div class="address-section">
 		<h1 class="has-margin-top-36 title is-size-5">{{ $t( 'membership_form_section_address_title' ) }}</h1>
```

Comment:
Move this into the `AddressPage` component, and you can remove the has-margin-top class.